images/origin/Dockerfile

```diff
@@ -12,24 +12,20 @@ FROM openshift/origin-base
 
 COPY system-container/system-container-wrapper.sh /usr/local/bin/
 COPY system-container/config.json.template system-container/manifest.json system-container/service.template system-container/tmpfiles.template /exports/
-COPY bin/openshift /usr/bin/openshift
-RUN ln -s /usr/bin/openshift /usr/bin/oc && \
-    ln -s /usr/bin/openshift /usr/bin/oadm && \
-    ln -s /usr/bin/openshift /usr/bin/origin && \
-    ln -s /usr/bin/openshift /usr/bin/kubectl && \
-    ln -s /usr/bin/openshift /usr/bin/openshift-deploy && \
-    ln -s /usr/bin/openshift /usr/bin/openshift-recycle && \
-    ln -s /usr/bin/openshift /usr/bin/openshift-router && \
-    ln -s /usr/bin/openshift /usr/bin/openshift-docker-build && \
-    ln -s /usr/bin/openshift /usr/bin/openshift-sti-build && \
-    ln -s /usr/bin/openshift /usr/bin/openshift-f5-router && \
+RUN INSTALL_PKGS="origin" && \
+    yum install -y ${INSTALL_PKGS} && \
+    rpm -V ${INSTALL_PKGS} && \
+    yum clean all && \
     setcap 'cap_net_bind_service=ep' /usr/bin/openshift
```

Comment:
Isn't that true for Brew? /cc @tdawson and if it is, we should make it true for Origin